lib/rototiller/utilities/env_var.rb

```diff
@@ -0,0 +1,55 @@
+require 'rototiller/utilities/color_text'
+
+class EnvVar
+
+  include ColorText
+
+  attr_accessor :var, :message, :default
+
+  def initialize(var, message, default = nil)
+    # require 'pry'; binding.pry
+    @var = var
+    @message = message
+    # TODO: maybe see if the value was passed in the args array or not as a default (so that nil may be passed)
```

Comment:
i don't understand why you'd want to pass nil.  but either way default will just get set to nil, which is probably appropriate, and creates a required envvar, as far as i can tell...  tests seem to confirm this.